src/domain/transfer/transform.js

```diff
@@ -139,13 +130,13 @@ const toTransfer = (t) => {
   // TODO: Validate 't' to confirm if its from the DB transferReadModel or from the saveTransferPrepare
   if (t.isTransferReadModel) {
     Logger.debug('In aggregate transfer transform for isTransferReadModel')
-    return Util.filterUndefined(fromTransferReadModel(t)) // TODO: Remove this once the DB validation is done for 't'
+    return Util.omitNil(fromTransferReadModel(t)) // TODO: Remove this once the DB validation is done for 't'
   } else if (t.isSaveTransferPrepared) {
     Logger.debug('In aggregate transfer transform for isSaveTransferPrepared')
-    return Util.filterUndefined(fromSaveTransferPrepared(t)) // TODO: Remove this once the DB validation is done for 't'
+    return Util.omitNil(fromSaveTransferPrepared(t)) // TODO: Remove this once the DB validation is done for 't'
   } else if (t.saveTransferFulfilledExecuted) {
     Logger.debug('In aggregate transfer transform for isSaveTransferExecuted')
-    return Util.filterUndefined(fromSaveTransferExecuted(t)) // TODO: Remove this once the DB validation is done for 't'
```

Comment:
We should remove the Util.filterUndefined as the omitNil does the job already.

Reply:
Done. I thought we might have need it where we want to remove just undefined, not nulls.